Tests/SentryTests/Helper/SentrySerializationTests.swift

```diff
@@ -5,7 +5,7 @@ import XCTest
 class SentrySerializationTests: XCTestCase {
     
     private class Fixture {
-        static var invalidData = "hi".data(using: .utf8)!
+        static var invalidData = "hi".data(using: .utf8) ?? Data()
```

Comment:
`m`: Please take look at this swiftlint rule:

https://realm.github.io/SwiftLint/non_optional_string_data_conversion.html

It might be worth activating it, or at least change to use the newer syntax without optional handling.

Relevant details:
https://forums.swift.org/t/can-encoding-string-to-data-with-utf8-fail/22437/3

Reply:
TIL; updated it and will try to enable the SwiftLint rule.